src/components/ActionList/ActionList.scss

```diff
@@ -228,17 +228,7 @@ $active-indicator-width: rem(3px);
 
     // Only show when the button is selected, not active
     [aria-selected='true'] > &::before {
-      content: '';
-      position: absolute;
-      top: 0;
-      bottom: 0;
-      left: -1 * spacing(extra-tight);
-      height: 100%;
-      width: $active-indicator-width;
```

Comment:
You no longer use this variable within the file, and therefor it should be removed.